packages/opentelemetry-sdk-trace-base/test/common/export/BatchSpanProcessorBase.test.ts

```diff
@@ -517,4 +505,42 @@ describe('BatchSpanProcessorBase', () => {
       });
     });
   });
+
+  describe('Concurrency', ()=> {
+    it('should only send a single batch at a time', async () => {
```

Comment:
@dyladan Here's a test for the concurrency, let me know if you have differing conventions/strategies for these sorts of tests